R/TuneWrapper.R

```diff
@@ -74,8 +74,16 @@ trainLearner.TuneWrapper = function(.learner, .task, .subset = NULL,  ...) {
 
 #' @export
 predictLearner.TuneWrapper = function(.learner, .model, .newdata, ...) {
-  lrn = setHyperPars(.learner$next.learner, par.vals = .model$learner.model$opt.result$x)
```

Comment:
FYI I am relatively sure that removing this line (and not putting `.learner = lrn` further down etc) will break things.

Reply:
you mean `.learner$next.learner`? That moved to another line. But I added setHyperPars now again.